diesel_tests/tests/debug/mod.rs

```diff
@@ -1,15 +1,25 @@
+use schema::TestBackend;
 use diesel::*;
 
 #[test]
 fn test_debug_count_output() {
     use schema::users::dsl::*;
-    let sql = debug_sql!(users.count());
-    assert_eq!(sql, "SELECT COUNT(*) FROM `users`");
+    let sql = debug_sql::<TestBackend, _>(&users.count());
```

Comment:
This is the exact same test as https://github.com/diesel-rs/diesel/pull/1051/files#diff-b33826ad230edbe4ba2ad2925b396345R162
Not sure if it's worth keeping